cli/setup.py

```diff
@@ -80,8 +80,7 @@
     package_data={
         'dcoscli': [
             'data/*.json',
-            'data/help/*.txt',
-            'data/schemas/*.json'
```

Comment:
I think we need to have the /schemas/*.json . why was that removed?

Reply:
@kensipe there were three files inside the `schemas/` and two of them were used [here](https://github.com/dcos/dcos-cli/blob/a6b757fb12b8796a1b292d45cf11bf472e7e4833/cli/dcoscli/experimental/main.py#L162) and [here](https://github.com/dcos/dcos-cli/blob/a6b757fb12b8796a1b292d45cf11bf472e7e4833/cli/dcoscli/experimental/main.py#L186) in `dcos-cli/cli/dcoscli/experimental/main.py` Since I am removing the `dcos-cli/cli/dcoscli/experimental` with this PR i removed those files. I was not able to find any usages for `schemas/manifest-schema.json` 

Thoughts?

^ @larioj after you have added these files there were no modifications. Should we retain these? If so, can you help me understand how they are used?

Reply:
These schemas are used for validating the input and output of the build and migrate commands. Unless it is being used elsewhere, they should not be retained.